docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Sphinx==1.7.8
 sphinx-argparse==0.2.2
 sphinxcontrib-websupport==1.1.0
-sphinx_rtd_theme
+sphinx-rtd-theme==0.4.3
+sphinx-argparse==0.2.5
```

Comment:
this is duplicated by line `2`, i'd update line two to reflect the LATEST version, and ensure it builds locally